configs/common/packages.yaml

```diff
@@ -84,7 +84,8 @@
     g2c:
       require: '@1.6.4'
     g2tmpl:
-      require: '@1.10.2'
+      require:
+        - '@1.10.2'
```

Comment:
I don't really understand what's going on here; why wasn't this broken before?

Reply:
before we had "soft requirements"/suggestions like:
```
g2tmpl:
  version: "1.10.2"
```
The container recipe could "add" to this, but after changing it to
```
      require: '@1.10.2'
```
the container recipe couldn't override/add to this requirement. If you use the new form:
```
      require:
        - '@1.10.2'
```
you allow the container recipe again to add its additional requirements for the package